packages/augur-sdk/src/state/getter/Users.ts

```diff
@@ -229,7 +229,9 @@ export class Users {
                 ].balance
               );
             }
-            tradingPosition.position = rawPosition.toFixed();
+            tradingPosition.position = rawPosition
```

Comment:
I think I made a mistake on `position` the UI should use netPosition. `position` can be removed.